resources/production/chainspec.toml

```diff
@@ -329,6 +329,7 @@ dictionary_put = { cost = 9_500, arguments = [0, 1_800, 0, 520] }
 enable_contract_version = { cost = 200, arguments = [0, 0, 0, 0] }
 manage_message_topic = { cost = 200, arguments = [0, 30_000, 0, 0] }
 emit_message = { cost = 200, arguments = [0, 30_000, 0, 120_000] }
+generic_hash = { cost = 300, arguments = [0, 0, 0, 0, 0] }
```

Comment:
We'd put some weight to the input size ptr (I think this is 2nd element in an array?)